### ISSUE TITLE
ARIA 1.1 Multi-Select Listbox example and quirky browser/AT behaviour

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/aria-practices#1955]
The "Example 2: Multi-Select Listbox" example https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-rearrangeable.html seems to have some odd behaviour when it comes to announcing the checked/selected items in the list.

* Chrome/NVDA and Chrome/JAWS: non-selected options in the list are identified as not being selected, but selected/checked ones don't really convey their state (you have to infer that it's selected/checked by the fact that it does *not* announce it as not selected)
* Firefox/JAWS correctly announces the selected/checked ones as "selected". it also announces "checkmark" at the start (so e.g. "Checkmark Leather seats selected"
* Firefox/NVDA doesn't announce the fact that an option is not selected is announced. Selected/checked options aren't explicitly announced as such, *but* the checkmark is announced (e.g. "Check Leather seats")

Wondering if this flaky/inconsistent behaviour could be improved by perhaps using, as a pattern, `aria-checked` rather than `aria-selected` ? (and hiding the CSS-generated checkmark character by associating it with a `<span aria-hidden="true">` or similar, to make sure that some browser/AT combos don't redundantly announce it)

**Comment by JAWS-test** [comment]
> by perhaps using, as a pattern, aria-checked rather than aria-selected ?

The output would possibly be better with this, but I think aria-selected is better than aria-checked because aria-selected is the default attribute, which I think should be used primarily. For the inconsistent output, an bug ticket should be issued for the screen readers

> and hiding the CSS-generated checkmark character

I think that is important. Someone using the example and using a different icon can produce screen reader output that could be very confusing (depending on the icon used)
